### PR TITLE
[Term Entry] Dart Type Conversion .parse()

### LINKED ISSUE #4571: [Term Entry] Dart Type Conversion .parse()
### Reason/inspiration (optional)

We would like a new term entry under Dart for the Dart Type Conversion `.parse()` method.
Provide the entry in a markdown file under `docs/content/dart/concepts/type-conversion/terms/parse/parse.md`

The entry should
- Have an introductory paragraph about what the method is and does.
- A syntax section with a pseudo code block with an explanation of it.
- An example showing the method in use.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Mhidayah19**
### Description

- Added Dart Type Conversion .parse()
- Closes #4571

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by Sriparno08 — COMMENTED** [review]
Hey @Mhidayah19, thanks for contributing to Codecademy!

I've listed all the modifications that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Looks good for a second review 👍

File: content/dart/concepts/type-conversion/terms/parse/parse.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, the **`.parse()`** method is used to convert a value from a string to a specified [data type](https://www.codecademy.com/resources/docs/dart/data-types), such as an `int` or a `double`.
+
+## Syntax
+
+```pseudo
+int.parse(string)
```
Comment (letsbuildanything) [inline content/dart/concepts/type-conversion/terms/parse/parse.md:21]:
```suggestion
dataType.parse(string)
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@Mhidayah19 
I've requested below some changes. Please take a moment to address them. If you have any concerns or reasons to reject specific suggestions, feel free to discuss them.

**Comment by letsbuildanything** [comment]
@Mhidayah19  It's been more than two weeks since the request was made. Please proceed with implementing all the suggested modifications.
Thank you!

**Review by letsbuildanything — APPROVED** [review]
Ready to be Merged! 🚀

**Pull request merged**